src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateFromFormat = date.split(fromFormat[3]);
+  const dateToFormat = [];
+
+  for (let j = 0; j < 3; j++) {
+    for (let i = 0; i < 3; i++) {
+      if (fromFormat[i].includes(toFormat[j][0])) {
+        dateToFormat.push(dateFromFormat[i]);
+      }
+    }
+  }
+
+  for (let n = 0; n < 3; n++) {
+    if (fromFormat[n] === 'YY' && toFormat[n] === 'YYYY') {
+      if (+dateToFormat[n] < 30) {
```

Comment:
you can use Conditional (ternary) operator here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator